.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 npm-debug.log
 coverage
 .DS_Store
+package-lock.json
```

Comment:
Some more context:  https://yarnpkg.com/blog/2017/05/31/determinism/

I think it should be okay to check both in. We can do this in a follow-on.